vignettes/adsl.Rmd

```diff
@@ -241,19 +241,44 @@ is to be imputed to the first of the month, then set `date_imputation = "FIRST"`
 
 ### Disposition Status (e.g. `EOSSTT`) {#disposition_status}
 
-The function `derive_var_disposition_status()` can be used to derive a disposition status at a specific
+The function `derive_var_merged_cat()` can be used to derive a disposition status at a specific
 timepoint. The relevant disposition variable (`DS.DSDECOD`) is selected by adjusting the filter 
 parameter and used to derive `EOSSTT`.
 
-To derive the End of Study status (`EOSSTT`), a call could be:
+To derive the End of Study status (`EOSSTT`), this function must be provided with the mapping derivation for the `cat_fun` argument. The call below has the following mapping derivation:
+
+- `"NOT STARTED"` if `DSDECOD` is `"SCREEN FAILURE"` or `"SCREENING NOT COMPLETED"`
+- `"COMPLETED"` if `DSDECOD == "COMPLETED"`
+- `"DISCONTINUED"` if `DSDECOD` is not `"COMPLETED"` or `NA`
+- `"ONGOING"` otherwise
+
+a call could be:
+
+```{r eval=TRUE}
+# Setting up the values of the source variable as input for the cat_fun argument in derive_var_merged_cat function
+format_dsdecod <- function(dsdecod_cat) {
+  case_when(
+    dsdecod_cat %in% c("SCREEN FAILURE", "SCREENING NOT COMPLETED") ~ "NOT STARTED",
+    dsdecod_cat %in% "COMPLETED" ~ "COMPLETED",
+    dsdecod_cat != "COMPLETED" | is.na(dsdecod_cat) ~ "DISCONTINUED",
```

Comment:
I think it should be `!is.na(dsdecod_cat) ~ "DISCONTINUED",`. Otherwise, the last case is never used.